src/FeApp.style.js

```diff
@@ -69,7 +69,7 @@ export default css`
     padding: 8px;
     margin-top: 8px;
   }
-  .form--input--field input:focus {
+  .form--input--field :focus {
```

Comment:
why input is removed ?, if we do for the class directly, it will focus all ing-input fields itself

Reply:
Because focus was not getting applied to select field

Reply:
however focus will be same to all input fields na so i thought to remove and use it for both select and input